### PR TITLE
feature/1012 - Add KHOU

### LINKED ISSUE #1012: New airport: Houston William P. Hobby International Airport
William P. Hobby International airport is the secondary airport for Houston, TX, USA. In 2017, the airport handled 13,435,672 passengers. The four-runway airport is a focus city for Southwest Airlines

Completion checklist:
- [ ] polygonal airspace boundary
- [ ] all SIDs
- [ ] all STARs
- [ ] fixes for all approaches
- [ ] information to validate traffic levels
- [ ] base video map
- [ ] no console errors or warnings

### PR DESCRIPTION
**Pull request by JPmAn24**
Resolves openscope/openscope#1012 

The purpose of this pull request is to Add Houston Hobby Int'l Airport

### CODE REVIEW

**Comment by felixscheffer** [comment]
Don't forget to add the AIRAC property.

**Comment by JPmAn24** [comment]
I'm requesting a review just to see how I'm doing so far with this. Still not done though.

File: assets/airports/airportLoadList.js [hunk 1]
```diff
@@ -231,6 +231,13 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
         //     name: 'El Paso International Airport',
         //     premium: false
         // },
+        {
+            icao: 'khou',
+            level: 'easy',
+            name: 'Houston-Hobby Int\'l',
+            premium: false,
+            wip: false
```
Comment (Fechulo) [inline assets/airports/airportLoadList.js]: wip tag isn't needed anymore

File: assets/airports/khou.json [hunk 2]
```diff
@@ -0,0 +1,11022 @@
...
+                "SHLLL": ["SHLLL"]
+            },
+            "body": [
+                ["BAYYY", "A170+|A200-|S280"],
+                ["PUCKS", "A100+|A140-"],
+                ["FRDDY", "A80+|A100-|S240"],
+                ["FIGGG", "A60+|A70-"]
+            ],
+            "rwy": {
+                "KHOU4": [
```
Comment (Fechulo) [inline assets/airports/khou.json]: These should be all in one line per runway

File: assets/airports/khou.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 
 {
     
-    "airac": 1811,
+    "airac": 1901,
```
Comment (Fechulo) [inline assets/airports/khou.json]: Unless there have been any changes between yesterday and today, you can change this to 1902

File: assets/airports/airportLoadList.js [hunk 4]
```diff
@@ -235,8 +235,7 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
             icao: 'khou',
             level: 'easy',
```
Comment (Fechulo) [inline assets/airports/airportLoadList.js]: Current total acph is 13.68 which would fall under beginner level. If this value is still under 20 after the addition of GA spawnPatterns, the level should be changed to beginner.

File: assets/airports/khou.json (hunk 2 above)
Comment (Fechulo) [reply assets/airports/khou.json]: This still needs to be resolved

**Pull request closed without merging**